jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/media/impl/MediaServiceRestImpl.java

```diff
@@ -94,8 +108,7 @@ public Response getMediaTypes() {
         return ResponseBuilder.ok(mediaService.getMediaTypes());
     }
 
-    public void setMediaService(MediaService mediaService)
-    {
+    public void setMediaService(MediaService mediaService) {
```

Comment:
Why do we have this method?
If the mediaService isn't already autowired please use autowire and remove this setter.